man/org.freedesktop.hostname1.xml

```diff
@@ -101,8 +101,6 @@ node /org/freedesktop/hostname1 {
       readonly ay BootID = [...];
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly u VSockCID = ...;
-      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
-      readonly s ChassisAssetTag = '...';
```

Comment:
sorry, but this is wrong. i guess @bluca's testcase neutering is causing this.

i.e. if i revert c6a932fd0db5bfcc523d0a57351b906bce2955cd things work again.

Reply:
What testcase? What are you talking about?

Reply:
It's not that, the update-dbus-docs target doesn't have the required meson dependencies, I'll update the PR.